### PR TITLE
Added support for "--replicas-max-per-node"

### PR DESCRIPTION
**Pull request by derteufelqwe**
I added support for the parameter " --replicas-max-per-node", which got recently added in the api version 1.40.

### CODE REVIEW

**Comment by derteufelqwe** [comment]
Issue #1382

**Comment by derteufelqwe** [comment]
Bump?

**Comment by bsideup** [comment]
@derteufelqwe please sync with the latest master to fix the CI

File: docker-java-api/src/main/java/com/github/dockerjava/api/model/ServicePlacement.java [hunk 1]
```diff
@@ -54,4 +61,32 @@ public ServicePlacement withConstraints(List<String> constraints) {
...
+     *
+     * @param maxReplicas Max number of replicas
+     * @return This instance of ServicePlacement
+     */
+    public ServicePlacement withMaxReplicas(int maxReplicas) throws DockerClientException {
+        if (maxReplicas >= 0) {
+            this.maxReplicas = maxReplicas;
+
+        } else {
+            throw new DockerClientException("The Value for MaxReplicas must be greater or equal to 0");
```
Comment (bsideup) [inline docker-java-api/src/main/java/com/github/dockerjava/api/model/ServicePlacement.java:77]: please use `IllegalArgumentException`

Comment (derteufelqwe) [reply docker-java-api/src/main/java/com/github/dockerjava/api/model/ServicePlacement.java]: Good point. I've updated the PR.

**Pull request merged**